lib/open_mic.rb

```diff
@@ -0,0 +1,29 @@
+class OpenMic
+
+  attr_reader :location, :date, :performers
+
+  def initialize(info)
+    @location = info[:location]
+    @date = info[:date]
+    @performers = []
+  end
+
+  def welcome(user)
+    @performers << user
+  end
+
+  def repeated_jokes?
+    jokes_array = @performers.map do |user|
+      user.jokes
+    end.flatten
+
+
+    # Wondering if there might be a cleaner way to test for repeating jokes. I tried to look up some possible enumerables but wasn't having much luck, although I'm sure there is a way. Let me know what you think!
```

Comment:
This is really similar to what I would do, but a cleaner way would be to remove the `if` statement and just rely on the conditional statement to return the boolean you want: 
```
...
  jokes_arry.uniq.length < jokes_array.length
end
```